ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/144_dsc_negative_no_dependant_op.robot

```diff
@@ -25,36 +25,104 @@ ${IS_NOT_PRESENT}                           1
 
 
 *** Test Cases ***
-Validate DSC and DSCI Created Without Service Mesh And Serverless Operators     #robocop:disable
+Validate DSC and DSCI Created Without Service Mesh Operator     #robocop:disable
+    [Documentation]    The purpose of this Test Case is to validate that DSC and DSCI are created
+    ...                without Service Mesh Operator installer, but with errors
+    [Tags]    Operator    Sanity    Tier1    ODS-2584
+
+    Remove DSC And DSCI Resources
+    Uninstall Service Mesh Operator CLI
+
+    Log To Console    message=Creating DSCInitialization CR via CLI
+    Apply DSCInitialization CustomResource    dsci_name=${DSCI_NAME}
+    Log To Console    message=Checking DSCInitialization conditions
+    Wait Until Keyword Succeeds    6 min    0 sec
+    ...    Check DSCInitialization Missing Operator Conditions When Service Mesh Operator Is Not Installed
+
+    Log To Console    message=Creating DataScienceCluster CR via CLI
+    Apply DataScienceCluster CustomResource    dsc_name=${DSC_NAME}
+    Log To Console    message=Checking DataScienceCluster conditions
+    Wait Until Keyword Succeeds    6 min    0 sec
+    ...    Check DataScienceCluster Missing Operator Conditions When Service Mesh Operator Is Not Installed
+
+    [Teardown]    Reinstall Service Mesh Operator And Recreate DSC And DSCI
+
+Validate DSC and DSCI Created With Errors Without Service Mesh And Serverless Operators     #robocop:disable
     [Documentation]    The purpose of this Test Case is to validate that DSC and DSCI are created
     ...                without dependant operators ((servicemesh, serverless) installed, but with errors
-    [Tags]    Operator    Sanity    Tier1    ODS-2527
+    [Tags]    Operator    Tier1    ODS-2527
+
+    Remove DSC And DSCI Resources
+    Uninstall Service Mesh Operator CLI
+    Uninstall Serverless Operator CLI
 
     Log To Console    message=Creating DSCInitialization CR via CLI
     Apply DSCInitialization CustomResource    dsci_name=${DSCI_NAME}
-    Sleep  30s     reason=wait for 1 minute until DSCI is created and reconciled
+    Sleep  45s     reason=wait for 45 seconds until DSCI is created and reconciled
 
     Log To Console    message=Creating DataScienceCluster CR via CLI
     Apply DataScienceCluster CustomResource    dsc_name=${DSC_NAME}
+    Log To Console    message=Checking DataScienceCluster conditions
     Wait Until Keyword Succeeds    6 min    0 sec
-    ...    Check DataScienceCluster Conditions When Dependant Operators Are Not Installed
+    ...    Check DataScienceCluster Missing Operators Conditions When Dependant Operators Are Not Installed
 
     [Teardown]    Reinstall Service Mesh And Serverless Operators And Recreate DSC And DSCI
 
+Validate DSC and DSCI Created With No Errors Without Service Mesh And Serverless Operators When Kserve Is Unmanaged    #robocop:disable
```

Comment:
so you are not setting Kserve as Unmanaged, but it's Serverless actually which gets set to Unmanaged

Reply:
no no, I am setting kserve serving to Unmanaged, so the DataScienceCluster is created properly when both service mesh and serverless operators are not installed.

If kserve serving is setted to Managed and the operators are not installed, thats the case where the DSC is created with errors

Reply:
Kserve Serving is controlling Serverless operator

Reply:
let me check, I think I am missing a condition to check also the DSCInitialization. This one is the one which manages ServiceMesh

Reply:
done @bdattoma. Was missing a condition to set the DSCI service mesh to unmanaged as well